manim/_config/main_utils.py

```diff
@@ -111,7 +111,6 @@ def parse_args(args: list) -> argparse.Namespace:
     elif subcmd == "plugins":
         return _parse_args_plugins(args)
     elif args[1] == "--version":
-        print(f"Manim Community v{ __version__ }")
```

Comment:
Makes sense since it's printed in `__main__.py`, though I am curious why this did not print twice for everyone, only some people. Are you sure this won't break it for those people who this only printed once for?

Reply:
I think it should have printed twice for every one. No?

Reply:
Not for me.

Reply:
It slightly annoyed me while implementing the tests, so I removed this, and I think there should be no problem.

Reply:
I only got the message once prior to this change. Tested your change, and I am still getting 1 message so good for me.

Reply:
I don't understand why that happens though. I got straight two messages and removed one.

Reply:
Me neither. It's inconsistent between different computers

Reply:
In my case, it's printed twice when invoking `manim --version`

Reply:
that was something expected so I did so. There is also a test for checking this :)

Reply:
But this one shouldn't have been removed. This one should have: https://github.com/ManimCommunity/manim/blob/0b9b90d94eba8578d5f6c1841c1b5d4900b6e984/manim/__main__.py#L45

Reply:
> We shouldn't be printing out the version regardless of the command passed to manim.

I would like that to happen :)
I have seen many CLI interface do it for example [meson](https://mesonbuild.com) or chocolatey.

Reply:
Then I guess this is a fine behavior as the appropriate sys.exit() call is hidden below in this file. I think I'll have to update this in the click branch.